### PR TITLE
More Apply options for a face gear

### PR DESCRIPTION
**Pull request by hatcher36**
Ability to lower some masks down to remove encumberance while wearing em under the chin (or on the forehead). So you can wear your filter masks and other small facegear and put it on the mouth/eyes in one move when it needed. I think that would be cool.
Here is example for a welding goggles:
{
  "id": "goggles_welding",
  "type": "ARMOR",
  "name": "pair of welding goggles",
  "name_plural": "pairs of welding goggles",
  "description": "A dark pair of goggles.  They make seeing very difficult, but protect you from bright flashes. Use it, to lift to the forehead from the eyes.",
  "weight": 153,
  "volume": 1,
  "price": 1400,
  "to_hit": -3,
  "material": [ "plastic" ],
  "symbol": "[",
  "color": "dark_gray",
  "use_action": {
      "type": "transform",
      "msg": "You lifted your welding goggles to the forehead.",
      "target": "goggles_welding_lifted",
      "menu_text": "Flip"
  },
  "covers": [ "EYES" ],
  "coverage": 100,
  "encumbrance": 60,
  "warmth": 10,
  "material_thickness": 4,
  "environmental_protection": 6,
  "qualities": [ [ "GLARE", 2 ] ],
  "flags": [ "SUN_GLASSES", "FLASH_PROTECTION" ]
},
{
  "id": "goggles_welding_lifted",
  "type": "ARMOR",
  "name": "pair of welding goggles (lifted)",
  "name_plural": "pairs of welding goggles (lifted)",
  "description": "A dark pair of goggles.  They make seeing very difficult, but protect you from bright flashes.",
  "weight": 153,
  "volume": 1,
  "price": 1400,
  "to_hit": -3,
  "material": [ "plastic" ],
  "symbol": "[",
  "color": "dark_gray",
  "use_action": {
      "type": "transform",
      "msg": "You lower your welding goggles to the eyes.",
      "target": "goggles_welding",
      "menu_text": "Flip"
  },
  "covers": [ "HEAD" ],
  "coverage": 15,
  "encumbrance": 0,
  "warmth": 0,
  "material_thickness": 4,
  "environmental_protection": 0,
  "qualities": [ [ "GLARE", 2 ] ],
  "flags": [ "OUTER" ]
}

Btw, git request system is AWFUL omg.

### CODE REVIEW

**Comment by hatcher36** [comment]
Filter, dust, rebreathing, diving masks. Maybe gas mask, even if its really big, but its elastic and made from resin.
Ski, infrared, light amp, swimming goggles, safety glasses.

**Comment by Coolthulhu** [comment]
I'm not sure it's worth simulating. Having transformed items change occupied body part would lead to exploits like gas masks being put over beaks and a mountain of welding goggles stacked on head.

If it's just about wearing times, we could lower the time it takes to acquire goggles and just assume the player has any arbitrary goggles worn around neck and not deep in backpack.

**Comment by hatcher36** [comment]
Unequipped items occupying inventory slots and volume, weared items are not,  so, decreasing worn speed isn't a proper decision.
Ok, i get it, changing clothes body part covering is not a good idea. But mask could be moved into the OUTER mouth layer, with transorm to useless item with no encumberance and no env protection. That mask still will conflicting with a mutations.

**Comment by DangerNoodle** [comment]
> I'm not sure it's worth simulating. Having transformed items change occupied body part would lead to exploits like gas masks being put over beaks and a mountain of welding goggles stacked on head.

Can items even change what body parts they cover via transform?

**Comment by kevingranade** [comment]
Transformations are alowed to change the itype, so yes, you can change
literally anything about an item.  The only potentially iffy thing is
cachebusting, but I don't think we're heavily caching items.

**Comment by DangerNoodle** [comment]
Understood. I thought that I had read somewhere that cachebusting would cause problems if transforming armor into a different type of armor.

Minor issue discovered in testing is that removing the zero values on lifted goggles (encumbrance, warmth, etc) is now required to avoid load error (assignment does not change value).

**Comment by Coolthulhu** [comment]
If this is to go in, the lifted parts will all need to be rewritten using the "copy-from" syntax.

But I'm still not convinced that having this weird interaction exist is the right way to go.

File: noname.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "id": "goggles_welding",
+  "type": "ARMOR",
+  "name": "pair of welding goggles",
+  "name_plural": "pairs of welding goggles",
+  "description": "A dark pair of goggles.  They make seeing very difficult, but protect you from bright flashes. Use it, to lift to the forehead from the eyes.",
```
Comment (BrettDong) [inline noname.json]: > bright flashes. Use it, to lift to the forehead

Two spaces after a period.

File: noname.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "color": "dark_gray",
+  "use_action": {
+      "type": "transform",
+      "msg": "You lower your welding goggles to the eyes.",
+      "target": "goggles_welding",
+      "menu_text": "Flip"
+  },
+  "covers": [ "HEAD" ],
+  "coverage": 15,
+  "encumbrance": 0,
```
Comment (kevingranade) [inline noname.json]: Fairly major balance issue, this should still have significant encumbrance.

**Comment by kevingranade** [comment]
If you want this considered for merge it needs to actually apply to the repository, this just drops a json file in the root of the working directory where it will do nothing.  These definitions need to be added to an approporiate existing json file.

**Comment by kevingranade** [comment]
Non functioning PR, and no action to fix it in over a month.

**Pull request closed without merging**